html/captive-portal/lib/captiveportal/PacketFence/Controller/Signup.pm

```diff
@@ -5,7 +5,7 @@ use Date::Format qw(time2str);
 use pf::log;
 use pf::constants;
 use pf::config;
-use pf::temporary_password 1.11;
+use pf::password 2.01;
```

Comment:
I would just drop the version number to does not really give us anything and it is one more thing to maintain